src/routes/videoPlaybackProxy.ts

```diff
@@ -109,6 +113,12 @@ videoPlaybackProxy.get("/", async (c) => {
         "last-modified": googlevideoResponse.headers.get("last-modified") || "",
     };
 
+    if (title) {
+        headersForResponse["content-disposition"] = `attachment; filename="${
+            encodeURIComponent(title)
+        }"; filename*=UTF-8''${encodeRFC5987ValueChars(title)}`;
```

Comment:
wow, what a weird format. Nice work, good to have the link to the MDN doc in the comment on that other file :+1: